### ISSUE TITLE
How about cifar10 dataset?

### ISSUE THREAD

**Issue by ThinkLock** [RobRomijnders/ladder#1]
I just want to achieve the accuracy mentioned in his paper on cifar10 dataset using tensorflow, but unfortunately, I cannot get the result with about 80% accuracy, actually, only for the fully supervised learning with 4000 training examples cannot achieve the accuracy of 76%(mine is around 73%). Do you have any code that is available for the cifar10 training dataset using ladder networks? thank you~

**Comment by pribadihcr** [comment]
Hi @ThinkLock,

Did you try main_conv for MNIST? 
I got the nan value after several process:
At 24410 of 90000 acc 71.0(100.0) cost super 1.89( 0.03) unsuper 23.40( 23.77)
At 24420 of 90000 acc 72.0(100.0) cost super 2.10( 0.01) unsuper 20.95( 22.53)
At 24430 of 90000 acc 72.0(100.0) cost super 2.08( 0.02) unsuper 20.27( 21.95)
At 24440 of 90000 acc 72.0(100.0) cost super 2.25( 0.01) unsuper 27.48( 21.81)
At 24450 of 90000 acc 73.0(100.0) cost super 2.11( 0.01) unsuper 25.52( 21.87)
At 24460 of 90000 acc 10.0( 33.6) cost super nan( nan) unsuper nan( nan)
At 24470 of 90000 acc 10.0( 12.5) cost super nan( nan) unsuper nan( nan)
At 24480 of 90000 acc 10.0( 10.3) cost super nan( nan) unsuper nan( nan)
At 24490 of 90000 acc 10.0( 10.0) cost super nan( nan) unsuper nan( nan)
